.github/workflows/deploy.yml

```diff
@@ -2,7 +2,7 @@ name: deploy
 
 on:
   push:
-    branches: [main, dev]
+    branches: [main, dev, feat/network-statistics-endpoint]
```

Comment:
We should undo this before merging

Reply:
Done: 3d732c8b5f5671ca12423c63fc69fa0ec1fe050d